tests/Runtime/PhpFpmTest.php

```diff
@@ -745,6 +765,19 @@ public function test FPM timeouts are recovered from()
         }
     }
 
+    /**
+     * @see https://github.com/mnapoli/bref/issues/316
+     */
+    public function test large response()
```

Comment:
Sorry - out of curiosity. How come php doesn't fail with syntax error? Is it some UTF-8 character that "looks" like a space?

Reply:
:trollface: see https://mnapoli.fr/using-non-breakable-spaces-in-test-method-names/ for more details but it is valid PHP code as those are non-breaking spaces. This is an alternative to `snake_case`.

Reply:
Yup, nice one. Thanks 👍